### ISSUE TITLE
Fatal error when calling Zend\Mvc\DispatchListener::marshalBadControllerEvent() with a PHP 7 Throwable

### ISSUE THREAD

**Issue by acelaya** [zendframework/zend-mvc#181]
The method `Zend\Mvc\DispatchListener::marshalBadControllerEvent()` has an `\Exception` type hint on its last argument, but it is also called when a `\Throwable` like an `\Error` instance is caught.
That produces this Fatal error.

```
Fatal error: Uncaught TypeError: Argument 4 passed to Zend\Mvc\DispatchListener::marshalBadControllerEvent() must be an instance of Exception, instance of Error given
```

**Comment by michalbundyra** [comment]
Resolved in [Release 3.0.2](https://github.com/zendframework/zend-mvc/releases/tag/release-3.0.2). Please run `composer update` and try again.

**Comment by acelaya** [comment]
I see. Thanks :)
